release-1.8/release_notes_draft.md

```diff
@@ -44,3 +44,37 @@ please check out the [release notes guidance][] issue.
 ## **Deprecations**
 
 ## **Notable Features**
+
+### **Node Components**
+#### kubelet
```

Comment:
Also need to add and entry for hugepage support https://github.com/kubernetes/features/issues/275

```
[alpha]  Applications may now request pre-allocated hugepages by using the new `hugepages` resource in the container resource requests [#275](https://github.com/kubernetes/features/issues/275), [@derekwaynecarr](https://github.com/derekwaynecarr)
```

fyi @derekwaynecarr